src/main/java/InputData/uploadSongInputData.java

```diff
@@ -2,7 +2,17 @@
 
 public class uploadSongInputData {
     String filepath;
-    uploadSongInputData(String filepath){
+    String uploader;
```

Comment:
is the uploader the artist user's name? could you add a comment explaining what this string refers to/is used for?

Reply:
Yeah, username. It'd be used on my end for determining all the songs a user uploaded.